### DISCUSSION TITLE
Would it be possible to add ipfs-search to presearch?

### DISCUSSION THREAD

**Discussion by ghost** [PresearchOfficial/presearch-packages#170] [Q&A] [answered]
Hi everyone. 

I would like to know if it is possible to add ipfs inside the presearch. So... *Would it be possible to add [ipfs-search](https://ipfs-search.com/#/) to presearch?*

<br/>

**concept**
![image](https://user-images.githubusercontent.com/111701513/196533431-c2ae250d-b8fe-4656-a4c3-cf7f81d6cb5d.png)

**source-code:** [ipfs-search/ipfs-search](https://github.com/ipfs-search/ipfs-search):  *Search engine for the Interplanetary Filesystem.*

*Just a thought.*

**Comment by jejopl** [comment, marked as answer]
Hey @codehangen, I'll take a look on that and I'll see if we could implement it somehow. At the start, it could be a presearch package, triggered with `ipfs` word (if possible). 
I'll get back to you when I will have some more informations.

**Reply by ghost** [reply]
Hi jejopl, thank you for feedback.
